gedi/process_l2a.py

```diff
@@ -31,10 +78,8 @@ def gedi_L2A_to_df(
     names of the h5 files, and information about the desired output, ingest
     the h5 files and output a pd.DataFrame containing data for all the valid
     shots within the bounding box.
-
     The output will have an array-valued column that is designed to be handled
     by append_canopy_metrics and then deleted from the DataFrame.
-
     Parameters
     ----------
     file_dir : str
```

Comment:
The docstring here can be updated to remove `file_dir` kwarg since the function just takes full paths, instead of a path and filenames, now.